src/fidesops/ops/service/connectors/email_connector.py

```diff
@@ -46,5 +48,56 @@ def mask_data(  # type: ignore
         policy: Policy,
         privacy_request: PrivacyRequest,
         rows: List[Row],
+        input_data: Dict[str, List[Any]],
     ) -> Optional[int]:
-        """Stub"""
+        """Cache instructions for how to mask data in this collection.
+        One email will be sent for all collections in this dataset at the end of the privacy request execution.
+        """
+
+        manual_action: ManualAction = self.build_masking_instructions(
+            node, policy, input_data
+        )
+
+        logger.info("Caching action needed for collection: '%s", node.address.value)
+        privacy_request.cache_email_connector_template_contents(
+            step=CurrentStep.erasure,
+            collection=node.address,
+            action_needed=[manual_action],
+        )
+
+        return 0  # Fidesops itself does not mask this collection.
```

Comment:
This is a primary change of this PR - EmailConnector.mask_data caches some instructions on how to find and mask data for this particular collection.